docker-compose.yml

```diff
@@ -72,7 +72,7 @@ services:
       - ci
 
   a3p:
-    image: ghcr.io/agoric/agoric-3-proposals:latest
+    image: ghcr.io/agoric/agoric-3-proposals:latest@sha256:dbc0232a8ed567079aefe4f5de230298f28519a29347413bec79d9b47de37db2
```

Comment:
so it's more clear to the reader what the base is, please use the `use-upgrade-17` tag. That will also help when searching the org code for what images need updating when a new one is released.

please also update https://github.com/Agoric/agoric-3-proposals/blob/main/.github/pull_request_template.md with a note to file an issue on this repo to adopt the new proposal when it's added